api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/products_controller.rb

```diff
@@ -16,7 +16,8 @@ def index
       JSON.parse(params[:terms]).map { |t| @products = @products | MnoEnterprise::ProductMarkup.includes(DEPENDENCIES).where(Hash[*t]).where(active: true) }
       response.headers['X-Total-Count'] = @products.count
     else
-      query = MnoEnterprise::Product.apply_query_params(params).where(active: true).includes(DEPENDENCIES)
+      query = MnoEnterprise::Product.apply_query_params(params).where(active: true)
+      query = query.includes(DEPENDENCIES) unless params[:skip_dependencies]
```

Comment:
@ouranos @manu-d 
Shouldn't it be the front-end that would specify the included fields?
`query.includes(params[:includes])` so we can tune queries from the front-ends

Reply:
It would be even better to put it in the method apply_query_params (https://github.com/maestrano/mno-enterprise/blob/4.0/core/app/models/mno_enterprise/base_resource.rb#L28), but maybe as part as a bigger refactoring, since almost all the controllers act like this at the moment

Reply:
agreed